src/framework/preferences/html/PrefsEditorTemplate-lineSpace.html

```diff
@@ -1,5 +1,6 @@
 <div class="flc-prefsEditor-line-space">
     <h2><span class="fl-icon-line-space"></span><label class="flc-prefsEditor-line-space-label" for="line-space"></label></h2>
```

Comment:
Updated [https://issues.fluidproject.org/browse/FLUID-5417]FLUID-5417 description to include the label task and [https://issues.fluidproject.org/browse/FLUID-5678]FLUID-5678